OurUmbraco/Our/Models/UpgradeResultModel.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace OurUmbraco.Our.Models
+{
+    public enum UpgradeType
```

Comment:
I think this should have its own file

Reply:
The reason I kept it in this model file is because we are only using this enum in relation to UpgradeRrsuktModel! But let me know if you would still prefer it to be moved :)

Reply:
When it is only used for output, you can either have the enum as a nested class inside the same file or as a separate class (like now) in another file..

Reply:
Right! Will refactor it to be nested in the class 👍